Stratis.Bitcoin/Features/Consensus/CoinViews/FetchCoinsResponse.cs

```diff
@@ -2,24 +2,35 @@
 
 namespace Stratis.Bitcoin.Features.Consensus.CoinViews
 {
+    /// <summary>
+    /// Return value of <see cref="CoinView.FetchCoinsAsync(uint256[])"/>,
+    /// contains the coinview tip's hash and information about unspent coins in the requested transactions.
+    /// </summary>
     public class FetchCoinsResponse
     {
-		public FetchCoinsResponse()
-		{
+        /// <summary>Hash of the block header for which <see cref="UnspentOutputs"/> is related.</summary>
+        public uint256 BlockHash { get; set; }
 
-		}
-		public FetchCoinsResponse(UnspentOutputs[] unspent, uint256 blockHash)
-		{
-			this.BlockHash = blockHash;
+        /// <summary>Unspent outputs of the requested transactions.</summary>
+        public UnspentOutputs[] UnspentOutputs { get; set; }
+
+        /// <summary>
+        /// Parameterless constructor.
+        /// </summary>
+        /// <remarks>TODO: This does not seem to be used anywhere by anything, can we remove it?</remarks>
```

Comment:
I guess we can remove it (but not urgent so can be done when another PR is pushed)

Reply:
i'll create issue then